README.md

```diff
@@ -471,6 +471,47 @@ With the flag `--keep_going=N` Jazzer continues fuzzing until `N` unique stack t
 Particular stack traces can also be ignored based on their `DEDUP_TOKEN` by passing a comma-separated list of tokens
 via `--ignore=<token_1>,<token2>`.
 
+### Export coverage information
+
+The internally gathered JaCoCo coverage information can be exported in human-readable and JaCoCo dump format.
+These can help identify code areas that can not be reached and perhaps need improvements to make them more accessible
+for the fuzzer.
+
+The human-readable report contains coverage information, like branch and line coverage, on file level. It's useful to 
+get a quick overview about the overall coverage. The flag `--coverage_report=<file>` can be used to generate it.
+
+Similar to the JaCoCo `dump` command the flag `--coverage_dump=<file>` specifies a coverage dump file, often called
+`coverage.exec`, that should be generated after the fuzzing run. It contains a binary representation of the gathered
+coverage data in the JaCoCo format.
+
```

Comment:
I think we should provide a slightly more detailed description of what differs so that it's more likely people actually follow the suggestions. My attempt:

The JaCoCo `report` command can be used to generate reports based on this coverage dump. **Note:** The version
of the JaCoCo agent used by Jazzer internally differs slightly from the official one. As a result, a similarly modified version of the JaCoCo CLI tool has to be used to generate correct reports. The correct version is available at ....